### PR TITLE
Adding licenses

### PR DESCRIPTION
**Pull request by awjh-ibm**
Adding licenses requested in https://github.com/hyperledger/composer/issues/1220

Adding license checker to check these are in

### CODE REVIEW

**Comment by awjh-ibm** [comment]
Please do not merge until mark has approved the new license checker version

File: packages/composer-protobuf/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1070 @@
+{
```
Comment (caroline-church) [inline packages/composer-protobuf/package-lock.json]: Can you remove this file and add it to the git ignore please

File: packages/node-red-contrib-composer/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3552 @@
+{
```
Comment (caroline-church) [inline packages/node-red-contrib-composer/package-lock.json]: Can you also remove this file please

**Comment by awjh-ibm** [comment]
Package locks removed and ignored

**Pull request merged**